docs/misc/updatechecker.md

```diff
@@ -38,7 +38,7 @@ This is fairly self-explanatory, but some notes:
 * The changelog string can be separated into lines using `\n`. Some prefer to include a abbreviated changelog, then link to an external site that provides a full listing of changes.
 * Manually inputting data can be chore. You can configure your `build.gradle` to automatically update this file when building a release as Groovy has native JSON parsing support. Doing this is left as an exercise to the reader.
 
-- Some examples can be found here for [nocubes][], [Corail Tombstone][corail] and [Chisels & Bits 2][chisel].
+- Some examples can be found here for [nocubes][] and [Corail Tombstone][corail].
```

Comment:
Why not just use forge's promotions-slim.json as the example? Doesn't have a changelog, but its a good enough example for the basic use.